backend/services/mgmt/v1alpha1/job-service/jobs.go

```diff
@@ -1665,3 +1723,28 @@ func getJobSourceConnectionId(jobSource *mgmtv1alpha1.JobSource) (*string, error
 	}
 	return connectionIdToVerify, nil
 }
+
+func getConnectionSchemaConfigByConnectionType(connection *mgmtv1alpha1.Connection) (*mgmtv1alpha1.ConnectionSchemaConfig, error) {
```

Comment:
what is the purpose of this function? and why are all of the configs effectively just empty?